src/R/Components/Impl/PackageManager/Implementation/View/PackageList.xaml

```diff
@@ -83,18 +83,21 @@
                         </Button>
 
                         <!-- title & summary-->
-                        <TextBlock Grid.Column="2" FontWeight="Bold" TextTrimming="CharacterEllipsis" TextWrapping="NoWrap" VerticalAlignment="Center" Text="{Binding Path=Name, Mode=OneWay}"
+                        <StackPanel Grid.Column="2">
+                            <TextBlock FontWeight="Bold" TextTrimming="CharacterEllipsis" TextWrapping="NoWrap" VerticalAlignment="Center" Text="{Binding Path=Name, Mode=OneWay}"
                                    FontSize="{Binding ElementName=_self, Path=FontSize, Converter={x:Static wpf:Converters.FontScale122}}" Margin="4,0,0,0">
-                            <TextBlock.ToolTip>
-                                <ToolTip Visibility="{Binding Path=Title, Mode=OneWay, Converter={x:Static wpf:Converters.NullOrEmptyIsCollapsed}}">
-                                    <TextBlock Style="{StaticResource TooltipStyle}">
+                                <TextBlock.ToolTip>
+                                    <ToolTip Visibility="{Binding Path=Title, Mode=OneWay, Converter={x:Static wpf:Converters.NullOrEmptyIsCollapsed}}">
```

Comment:
Since title is already displayed in the list, I'd bind title to description. Although for the tooltip to look OK we would need to add line breaks every ~100 characters.